spk/synocli-fossil/Makefile

```diff
@@ -0,0 +1,24 @@
+SPK_NAME = synocli-fossil
+SPK_VERS = 1.0
+SPK_REV = 3
+SPK_ICON = src/synocli-fossil.png
+
+include ../../mk/spksrc.common.mk
```

Comment:
remove include of spksrc.common.mk, needed only when common variables are used before include of spksrc.spk.mk (like xxx_ARCHES)